resources/views/layouts/app.blade.php

```diff
@@ -314,6 +322,104 @@
+            success: function (data) {
+                console.log(data);
+
+                var content = "";
+                if (data.cart_count > 0) {
+                    content = '<div class="all-cart-product clearfix">';
+
+                    for (index in data.items) {
+                        var item = data.items[index];
+
+                        content += '' +
+                            '<div class="single-cart clearfix">\n' +
+                            '   <div class="cart-image">\n' +
+                            '       <a href="/product/'+item.slug+'">' +
+                            '           <img src="/products/images/'+item.attributes.img+'" alt=""></a>\n' +
+                            '   </div>\n' +
+                            '   <div class="cart-info">\n' +
+                            '       <h5><a href="/product/'+item.slug+'">' +
+                            '               '+ item.name +'</a></h5>\n' +
+                                '   <p>Price : $'+ item.price.toFixed(2) +'</p>\n' +
+                                '   <p>Qty : '+ item.quantity +'</p>\n' +
+                                '   <a href="#" class="cart-delete" title="Remove this item"><i class="pe-7s-trash"></i></a>\n' +
+                            '   </div>\n' +
+                            '</div>';
+                    }
+
+                    content += '' +
+                        '</div>' +
+                        '' +
+                        '<div class="cart-totals">' +
+                        '   <h5>Total <span>$'+ data.total.toFixed(2) +'</span></h5>' +
+                        '</div>' +
+                        '\n' +
+                        '<div class="cart-bottom">\n' +
+                        '   <a href="{!! route('cart') !!}" class="btn-sm">Cart</a>\n' +
+                        '   <p class="clearfix"></p>\n' +
+                        '   <a href="{!! route('checkout') !!}" class="btn-sm">Check out</a>\n' +
+                        '</div>';
+                } else {
+                    content = '' +
+                        '<div class="all-cart-product clearfix">\n' +
+                        '<img src="{!! asset('img/empty_cart.png')  !!}" class="img-responsive center-block" style="margin-bottom: 15px">\n' +
+                        '</div>\n' +
+                        '\n' +
+                        '<!-- Cart Button -->\n' +
+                        '<div class="cart-bottom">\n' +
+                        '<a href="{!! route('shop') !!}" class="btn-sm">Continue Shopping</a>\n' +
+                        '</div>';
+                }
+
+                $("#cart_product_list").html(content);
+
+            },
+            error: function (xhr, status, error) {
+                console.log(xhr.responseText);
+                alert(error.msg);
+            }
+        });
+
+        //TODO make a request to the server to get all items on the cart
+        //TODO Update the cart_product_list
```

Comment:
I think its is okay.. Another way to load the cart in all views is to use ( a variable in laravel that can be shared amongst all views). Check on that or we could still go this way which is stressful IMO. Anyway you choose aside that all is good to go

Reply:
I don't understand what you mean by using a variable that can be shared in all the views